packages/Input/src/Input.js

```diff
@@ -4,25 +4,76 @@ import classNames from "classnames";
 import * as constants from "@paprika/constants/lib/Constants";
 import TimesCircleIcon from "@paprika/icon/lib/TimesCircle";
 import Button from "@paprika/button";
+import useI18n from "@paprika/l10n/lib/useI18n";
 import * as types from "./types";
 import * as sc from "./Input.styles";
 
+function callAll(...fns) {
```

Comment:
Maybe this should be added to `@paprika/helpers` to be shared?

Reply:
wouldn't hurt, up to you :)